modules/stitching/include/opencv2/stitching.hpp

```diff
@@ -193,6 +193,9 @@ class CV_EXPORTS_W Stitcher
     CV_WRAP bool waveCorrection() const { return do_wave_correct_; }
     CV_WRAP void setWaveCorrection(bool flag) { do_wave_correct_ = flag; }
 
+    CV_WRAP int interpolation() const { return interp_mode_; }
```

Comment:
> interpolation

getInterpolation / getInterpolationFlags

Reply:
I replaced `interpolation` by `getInterpolationFlags` and `setInterpolation` by `setInterpolationFlags`, and renamed the attribute accordingly.
I notice that all other getters of the Stitcher are named `attribute()` and not `getAttribute()`, is this intentional ?

Reply:
Looks like you are right!

This is not aligned with other OpenCV parts, but lets keep stitching API consistent.

Renamed to `interpolationFlags()`